app/controllers/sessions_controller.rb

```diff
@@ -47,7 +47,7 @@ def create
       @current_user.update_attribute(:facebook_id, session["devise.facebook_data"]["id"])
       # FIXME: Currently this doesn't work for very unknown reason. Paper clip seems to be processing, but no pic
       if @current_user.image_file_size.nil?
-        @current_user.store_picture_from_facebook
+        @current_user.store_picture_from_facebook!
```

Comment:
I didn't add exception catching here, because I didn't quite understand when this piece of code is executed (if ever? 😬)